drivers/axi_core/axi_adc_core/axi_adc_core.c

```diff
@@ -150,6 +150,118 @@ void axi_adc_idelay_set(struct axi_adc *adc,
 
 }
 
+/***************************************************************************//**
+ * @brief axi_adc_delay_set
```

Comment:
Since this is an API I think we will benefit from more descriptive comments.

Reply:
This approach was used here to keep the consistency with the rest of the file (all functions in `axi_adc_core` have short descriptions like this one). I believe adding proper header comments to the entire file is subject for another commit.